mkdocs.yml

```diff
@@ -16,20 +16,20 @@ nav:
   - Roadmap: roadmap.md
   - Related projects: related.md
   - API Reference:
-    - api-reference/index.md
```

Comment:
thanks - i think all the upper-case ones should go first, and then the lowercase ones?

```
In [10]: sorted('DLdl')
Out[10]: ['D', 'L', 'd', 'l']
```